src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -183,6 +195,14 @@ void initializeReference() {
         reference = referenceArguments.getReferenceFile() != null ? ReferenceDataSource.of(referenceArguments.getReferenceFile()) : null;
     }
 
+    private static SeekableByteChannel addPrefetcher(int cloudPrefetchBuffer, SeekableByteChannel x) {
+        try {
+            return new SeekableByteChannelPrefetcher(x, cloudPrefetchBuffer * 1024 * 1024);
+        } catch (IOException ex) {
+            throw new GATKException("Unable to initialize the prefetcher: " + ex);
+        }
+    }
```

Comment:
As mentioned in another comment that might have gotten buried/rebased away, please do move all this code in `GATKTool` down into `ReadsDataSource`, and have the `ReadsDataSource` constructors take the buffer sizes as ints. There are clients in gatk-protected that use `ReadsDataSource` directly and would benefit, and in general we want to encapsulate these sort of low-level details in the data source classes as much as possible, and not bleed into the tool hierarchy.